package.json

```diff
@@ -76,8 +76,8 @@
     "kafka-node": "^1.0.7",
     "lodash": "^4.17.4",
     "mocha": "2.3.3",
-    "mongodb": "2.2.33",
-    "mongoose": "4.7.1",
+    "mongodb": "3.1.4",
+    "mongoose": "5.2.13",
```

Comment:
Testing with latest versions on Travis but I tested it manually with the versions we used previously. I guess having automated tests with multiple package versions would be awesome, but quite a lot of effort to build.

Reply:
This is something that has been on my mind as well for quite some time. We already have instrumentations which only work in specific versions, but we aren't properly testing these. Would be great to attempt such a test harness rather sooner than later.